Formula/opencv@2.rb

```diff
@@ -66,6 +66,7 @@ def install
 
     mkdir "build" do
       system "cmake", "..", *args
+      inreplace "modules/core/version_string.inc", "#{HOMEBREW_SHIMS_PATH}/mac/super/", ""
```

Comment:
Does this still use shims at build-time?

Reply:
Same ⬆️